### ISSUE TITLE
Feature Request: Journal Entry Template

### ISSUE THREAD

**Issue by GoodGuyPat** [leenzhu/joplin-plugin-journal#4]
Hi!
Thank you for releasing this plugin, will be very useful!
Most of people who journal follow the same structure on all their entries (as in, dividing it in sections). It would be useful to be able to create a template that will be used as the body of new notes, when created.

**Comment by TheHuntyBadger** [comment]
I've been using this plugin to create the notes and another plugin to insert Templates into the Day's notes:

https://github.com/joplin/plugin-templates

You can insert several templates if you have different sections to your journaling. it would be nice to have option for 'blank' or 'from template'

**Comment by knas4ever** [comment]
yes ability to pick template for journal creation would be very helpful.
thanks in advance for considering this feature.

**Comment by leenzhu** [comment]
Please check the v2.0.0 for trying template 😄
